tests/forms/App.xaml.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+using Xamarin.Forms;
+
+namespace forms
+{
+    public partial class App : Application
```

Comment:
I take it that the default Xamarin.Forms template uses spaces? Not sure if this needs to be fixed, just seems weird.